spec/models/froggo_team_membership_spec.rb

```diff
@@ -5,6 +5,11 @@
   let!(:organization) { create(:organization) }
   let!(:team) { create(:froggo_team, organization: organization) }
 
+  describe "validations" do
+    it { is_expected.to validate_numericality_of(:assignment_percentage)
+      .only_integer.is_greater_than_or_equal_to(1).is_less_than_or_equal_to(100) }
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Layout/BlockEndNewline: Expression at 10, 82 should be on its own line.